apache/zeppelin
# [ZEPPELIN-2025] Cleanup R Tutorial

### What is this PR for?

1. Remove useless paragraphs in R tutorial
2. Prettify layout

I attached screenshots for comparison. See the screenshot section below.

### What type of PR is it?
[Improvement]

### Todos

Done

### What is the Jira issue?

[ZEPPELIN-2025](https://issues.apache.org/jira/browse/ZEPPELIN-2025)

### How should this be tested?

1. Run Zeppelin
2. See the R tutorial


### Screenshots (if appropriate)

#### Before

![before](https://cloud.githubusercontent.com/assets/4968473/22393959/5266c76a-e556-11e6-8501-34d87c31908e.gif)

#### After

![after](https://cloud.githubusercontent.com/assets/4968473/22393961/5394bc50-e556-11e6-83f8-23ccaa05c12a.gif)

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO
